### PR TITLE
switches database to postgres

### PR DESCRIPTION
**Pull request by kenodressel**
there are steps that have to be done before we can merge this #166

### CODE REVIEW

File: logic/commentLogic.js [hunk 1]
```diff
@@ -13,12 +13,13 @@ module.exports = class CommentLogic {
       if (tipId === null || tipId === undefined || !text || !author || !signature || !challenge) {
         return res.status(400).send('Missing required field');
       }
-
-      const parentComment = typeof parentId !== 'undefined'
+      const parentComment = (typeof parentId !== 'undefined' && parentId !== '')
         ? await Comment.findOne({ where: { id: parentId } }, { raw: true }) : null;
-      if (parentComment === null && typeof parentId !== 'undefined') return res.status(400).send(`Could not find parent comment with id ${parentId}`);
+      if (parentComment === null && typeof parentId !== 'undefined' && parentId !== '') {
```
Comment (thepiwo) [inline logic/commentLogic.js:18]: can it be all, `null`, `undefined` or `''`? Otherwise let's only do the relevant checks

**Review by thepiwo — APPROVED** [review]
lgtm

File: logic/commentLogic.js (hunk 1 above)
Comment (kenodressel) [reply logic/commentLogic.js:18]: it technically should not even be an empty string, its only a bug in the currently deployed frontend

Comment (thepiwo) [reply logic/commentLogic.js:18]: ok, then lets keep as is

**Review by thepiwo — APPROVED** [review]

**Pull request merged**